app/src/androidTest/java/org/simple/clinic/benchmark/BackupBenchmarkDatabase.kt

```diff
@@ -0,0 +1,37 @@
+package org.simple.clinic.benchmark
+
+import android.app.Application
+import java.io.File
+import javax.inject.Inject
+
+class BackupBenchmarkDatabase @Inject constructor(
+    private val application: Application,
+    private val database: org.simple.clinic.AppDatabase
+) {
+
+  fun backup() {
+    val databaseDirectory = File(database.openHelper.readableDatabase.path).parentFile!!
+    val databaseBackupDirectory = resolveDatabaseBackupDirectory()
```

Comment:
If these don't change, we can move them to class `init`?

Reply:
I vaguely remember trying to do that, but there was an issue because the open helper changes across test runs so the original file handle is not valid and throws an exception. Something that can be looked at later, I feel.